### PR TITLE
Added domains display

### PR DESCRIPTION
**Pull request by Bandius**
In tenant admin, domains connected to tenant are shown in Tenant Profile.

### CODE REVIEW

File: frontend/admin/src/components/Tenant/TenantShow.tsx [hunk 1]
```diff
@@ -65,14 +65,28 @@ const TenantShow: (props: ShowProps) => JSX.Element = (props) => {
           reference="users"
           target="tenantId"
           sort={{ field: 'id', order: 'ASC' }}
+          filter={{ type: 'email' }}
         >
           <Datagrid>
-            <TextField source="id" />
+            {/* <TextField source="id" /> */}
```
Comment (matusziak) [inline frontend/admin/src/components/Tenant/TenantShow.tsx:71]: Remove this forgotten comment.

File: frontend/admin/src/components/Tenant/TenantShow.tsx [hunk 2]
```diff
@@ -65,14 +65,28 @@ const TenantShow: (props: ShowProps) => JSX.Element = (props) => {
...
+          reference="users"
+          target="tenantId"
+          sort={{ field: 'id', order: 'ASC' }}
+          filter={{ type: 'domain' }}
+        >
+          <Datagrid>
+            <TextField source="name" />
+            <EditButton />
+          </Datagrid>
+        </ReferenceManyField>
```
Comment (matusziak) [inline frontend/admin/src/components/Tenant/TenantShow.tsx:89]: Domain should be a resource that we store in the database for each tenant. Not a calculated field based on the users' emails. Please add it to the mock data and models. And communicate if the backend is aware of how this should work.

**Review by matusziak — CHANGES_REQUESTED** [review]

File: frontend/admin/src/components/Tenant/TenantShow.tsx (hunk 2 above)
Comment (Bandius) [reply frontend/admin/src/components/Tenant/TenantShow.tsx:89]: Problem is that mock, doesn't allow to have more than 5 objects, which we currently have. This is why I chose this approach. I was trying to do it that way.

Comment (matusziak) [reply frontend/admin/src/components/Tenant/TenantShow.tsx:89]: I see. How is the BE state, could we switch and basically start integration? And for testing BE non-existing features we could still switch back in the code with the URL.

**Pull request closed without merging**